src/lib/components/forms/form/form-components/color-picker/color-picker.component.spec.ts

```diff
@@ -123,4 +123,24 @@ describe('ColorPickerComponent', () => {
 
         expect(spy).toHaveBeenCalled();
     });
+
+    it('should display an error message when pattern does not match', async () => {
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+
+        component.writeValue('#ffffff');
```

Comment:
Ich würde empfehlen hier überall eher auf `input.setValue()` zu setzen. Das simuliert nämlich tatsächlich eine Nutzereingabe.
```suggestion
        await input.setValue('#ffffff');
```